### PR TITLE
chore: upgrade portal to fix partials

### PR DESCRIPTION
**Pull request by RomanHotsiy**
Upgrades portal to fix partials.

Also, fixes some small ui issues.

### CODE REVIEW

**Comment by RomanHotsiy** [comment]
@amarantha-k I fixed it.

**Comment by amarantha-k** [comment]
@RomanHotsiy The footer style is altered in the PR (screenshot below) and doesn't match the style on the site currently. 
  
Screenshot 2023-05-30 at 12 36 01 PM

**Comment by RomanHotsiy** [comment]
Oh, yes. It was like that before too I believe (I mean last time you attempted to publish).

I can adjust it later today.

File: redocly.yaml [hunk 1]
```diff
@@ -32,20 +39,27 @@ theme:
     copyrightText: >-
       © 2013 - 2023 Ripple, All Rights Reserved.
     items:
-      - group: About
+      - group: Support
```
Comment (amarantha-k) [inline redocly.yaml:42]: The footer content should not be updated as opensource.ripple.com is a separate site and we don't want to add Ripple Enterprise product links in the footer. 
Just need the same list of links that we see on the site currently (screenshot attached)
Screenshot 2023-07-27 at 4 02 22 PM

**Review by amarantha-k — CHANGES_REQUESTED** [review]

File: redocly.yaml (hunk 1 above)
Comment (RomanHotsiy) [reply redocly.yaml]: Resolved

**Comment by amarantha-k** [comment]
The changes look good. Thanks for fixing these issues.

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**